map/bookmark_manager.cpp

```diff
@@ -2495,8 +2546,14 @@ bool BookmarkManager::DeleteBmCategory(kml::MarkGroupId groupId, bool deleteFile
   ClearGroup(groupId);
   m_changesTracker.OnDeleteGroup(groupId);
 
-  if (deleteFile)
-    FileWriter::DeleteFileX(it->second->GetFileName());
+  auto const filePath = it->second->GetFileName();
```

Comment:
Should it be
```suggestion
  auto const & filePath = it->second->GetFileName();
```
to avoid unnecessary allocation/copy?

Reply:
It's not about $$$. It's about clarity and clear code. And understanding what's going on.